### ISSUE TITLE
Multiple browserify instances with same dependencies causes missing CSS

### ISSUE THREAD

**Issue by huan086** [css-modules/css-modulesify#103]
Bundle1: A.js depends on A.css
Bundle2: B.js depends on A.js

Browserify transform only calls transform on A.css **once**. Due to loader.tokensByFile populated during transform phase, the tokens are loaded into Bundle1's loader. Bundle2's loader thus do not have the tokens.

It looks like the correct place to get dependencies is `bundle.pipeline.get('deps')`

**Comment by joshwnj** [comment]
Thanks for reporting this @huan086 . If you'd like to try creating a failing test case that'd be helpful too

**Comment by huan086** [comment]
Hmm, can't reproduce with browserify and css-modules alone. Might be some plugins having conflicts, let me investigate further.

**Comment by huan086** [comment]
Sorry, it wasn't a problem with modulesify. Found the root cause to be how I've specified my options.

I'm using multiple browserify instances with watchify. I've initialized browserify with

`browserify(files, assign({}, watchify.args, { plugin: browserifyPlugins }))`

Specifying `watchify.args` causes all the browserify instances to share the same `cache: {}, packageCache: {}`, causing browserify to go haywire, e.g. calling transform only once for all the instances.
